developer_manual/core/integration-tests.rst

```diff
@@ -0,0 +1,167 @@
+Integration tests
+============
+
+Using Behat to test core 
+----------------
+
+Tests location and structure
+~~~~~~~~~~~~~~~~~~~~~~
+
+In core repository we can find this structure:
+
+.. code-block:: bash
+    build
```

Comment:
please move this to "tests/integration" as part of your work that makes apps use that folder when running their own integration tests.